sklearn/linear_model/logistic.py

```diff
@@ -711,10 +710,20 @@ def logistic_regression_path(X, y, pos_class=None, Cs=10, fit_intercept=True,
     n_iter = np.zeros(len(Cs), dtype=np.int32)
     for i, C in enumerate(Cs):
         if solver == 'lbfgs':
+            if 0 < verbose <= 1:
+                iprint = 50
+            elif 1 < verbose <= 2:
+                iprint = 1
+            elif 2 < verbose <= 3:
+                iprint = 100
+            elif verbose > 3:
+                iprint = 101
+            else:
+                iprint = -1
```

Comment:
you should be able to do it in one line with `np.searchsorted` so you don't decrease the coverage without adding a dedicated test.

Reply:
such as `[-1, 50, 1, 100, 101][np.searchsorted(np.array([0,1,2,3]), 4)]`?